Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM python:3.7-slim
+MAINTAINER numigi <contact@numigi.com>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [MAINTAINER is deprecated](https://app.codacy.com/app/numigi/gitoo/pullRequest?prid=4550131)